# Contains a move on the board (col, row) and player (0 -> O, 1 -> X)
uint8 col
uint8 row
uint8 player 